samples/lazylist.golo

```diff
@@ -0,0 +1,55 @@
+# ............................................................................................... #
```

Comment:
I'll fix that one: for samples we should rather use the Eclipse Distribution License (it's a BSD, really)